### PR TITLE
Clean up: research product version

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- establish alphabetical order
- establish correct order within the properties

MAJOR changes:
none

SPECIAL ATTENTION:
- removed maxLength for both description & shortName because I don't believe that we adhered to this so far and I don't see a good reason to have it here anymore
- removed the notion that no references are allow from the instructions of the description

CHANGELOG:
- `homepage` asks for IRI now instead of linking to `URL`
- `fullDocumentation` links to `WebResource` instead of `URL`

- adjustment of relatedPublication to fit the use for `livePaperVersion`:
     - add `ISSN` as allowed value
     - adjusted instructions to include the use of this property from the live paper version (original instruction there: "Add an identifier for the book, chapter, article, or preprint for which this live paper provides supplementary material.")

### CODE REVIEW

File: schemas/products/researchProductVersion.schema.tpl.json [hunk 1]
```diff
@@ -69,11 +68,11 @@
       "_instruction": "Enter the preferred citation text for this research product version. Leave blank if citation text can be extracted from the assigned digital identifier."
     },
     "keyword": {
-      "type": "array",
-      "maxItems": 5,
+      "type": "array",      
       "minItems": 1, 
+      "maxItems": 5,
```
Comment (lzehl) [inline schemas/products/researchProductVersion.schema.tpl.json]: please remove this as well. if remove other restrictions we should not introduce one here. Each system (e.g. EBRAINS KG can decide this for themselves)

Comment (lzehl) [reply schemas/products/researchProductVersion.schema.tpl.json]: adapt then again the instructions as well

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/products/researchProductVersion.schema.tpl.json (hunk 1 above)
Comment (UlrikeS91) [reply schemas/products/researchProductVersion.schema.tpl.json]: Agreed and updated 🙂

File: schemas/products/researchProductVersion.schema.tpl.json [hunk 2]
```diff
@@ -90,11 +90,12 @@
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add all further publications besides the full documentation that provide the original context for the production of this research product version (e.g., an original research article).",
+      "_instruction": "Add all further publications besides the full documentation that provide the original context for the production of this research product version (e.g., an original research article) or receive supplementary material by this research product version.",
```
Comment (lzehl) [inline schemas/products/researchProductVersion.schema.tpl.json]: I do not understand the addition. And "supplementary material" would include also learning resources which is explicitly not what is meant here.

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/products/researchProductVersion.schema.tpl.json [hunk 3]
```diff
@@ -91,19 +90,20 @@
...
       ]
     },
     "releaseDate": {
+      "type": "string",
       "_formats": [
         "date"
       ],
-      "type": "string",
-      "_instruction": "Enter the date (actual or intended) of the first broadcast/publication of this research product version."
+      "_instruction": "Enter the date (actual or intended) on which this research product version was first release, formatted as '2023-02-07'."
```
Comment (aeidi89) [inline schemas/products/researchProductVersion.schema.tpl.json]: Suggest changing '2023-02-07' to 'YYYY-MM-DD'

**Review by lzehl — APPROVED** [review]
@aeidi89 I've updated the schema with your issue and adapted the instructions a bit of the "relatedPublication" property. Please double check and feel free to merge if you approve the changes.

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**